theme/update_theme.rs

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env cargo +nightly -Zscript
+
+// This script fetches the 'index.hbs' file from the mdBook repository and filters the
+// content to only present the 'ayu' theme.
+
+//! ```cargo
+//! [dependencies]
+//! reqwest = { version = "0.11", features = ["blocking"] }
+//! ```
+
+use reqwest::blocking;
+use std::fs::File;
+use std::io::{self, Write};
+
+const MDBOOK_VERSION: &str = "0.4.36";
+
+fn main() -> io::Result<()> {
+    let url = format!(
+        "https://raw.githubusercontent.com/rust-lang/mdBook/v{MDBOOK_VERSION}/src/theme/index.hbs"
+    );
+
+    let response = blocking::get(&url)
+        .expect("Failed fetch template file")
```

Comment:
the message should be what you expect to happen, ref here https://github.com/tlsnotary/tlsn/pull/392#discussion_r1420600955